libnavui-dropin/src/main/res/layout-land/mapbox_navigation_view_layout.xml

```diff
@@ -129,29 +129,23 @@
 
             </androidx.constraintlayout.widget.ConstraintLayout>
 
-            <LinearLayout
+            <FrameLayout
                 android:id="@+id/infoPanelLayout"
-                android:orientation="vertical"
                 android:layout_width="375dp"
                 android:layout_height="wrap_content"
-                android:elevation="8dp"
-                android:background="@drawable/mapbox_bg_info_panel"
                 app:layout_behavior="com.google.android.material.bottomsheet.BottomSheetBehavior"
                 app:behavior_hideable="true"
                 app:behavior_fitToContents="true"
-                app:behavior_peekHeight="@dimen/mapbox_infoPanel_peekHeight">
+                app:behavior_peekHeight="@dimen/mapbox_infoPanel_peekHeight"
```

Comment:
We don't need it anymore since we are assigning peek height at runtime. I removed it on your branch locally and it's working fine.